CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 # Change Log
 
-## v0.0.2-beta
+## 1.0.0
+
+**Release date:** 2022-01-31
+
+* Improved documentation
+* Changed the defualt values in configuration template
+* Resume the Bamboo server after installation
+* Improved install and uninstall scripts
+* Made the product version, Bamboo software and Bamboo agent resources configurable
+* Improved the end to end tests
+* Changed the default product version (Bamboo software and Bamboo agent Helm charts) to `1.0.0`
+* Changed the maximum length for environment name to 24 characters
```

Comment:
We also now restrict the environment to lowercase letters as well.

Reply:
done